L1Trigger/TrackFindingTracklet/test/L1TrackNtupleMaker_cfg.py

```diff
@@ -169,6 +169,8 @@ def getTxtFile(txtFileName):
                                        # tracking in jets (--> requires AK4 genjet collection present!)
                                        TrackingInJets = cms.bool(False),
                                        GenJetInputTag = cms.InputTag("ak4GenJets", ""),
+                                       # Was track quality performed?
+                                       TrackQuality = cms.bool(False)
```

Comment:
If the current default model is not included, then perhaps we should include a comment here with a link to where to find this model. Something like "Is track quality performed? (--> requires model in L1Trigger/TrackTrigger/data present! default: https://github.com/Chriisbrown/L1Trigger-TrackTrigger)". Is there a better place to include this link?

Reply:
I have a  comment in the TrackQualityParams_cfi.py file but might also be worth having the same comment in the Ntuple maker too?